File: Dw.ClickHouse/Dtos/OrderBy/ChOrderByDto.cs
using Ch.Models.Dtos.Output;
using Ch.Models.Enums.Order;

namespace Ch.Models.Dtos.OrderBy
{
    public class ChOrderByDto
    {
        public ChColumnOutputDto OrderByColumn { get; set; }
        public OrderByType OrderBy { get; set; } = OrderByType.Asc;
    }
}

File: WebApplication/src/app-clickhouse/dtos/order-by/ch-order-by.dto.ts
import { OrderByType } from "../../enums/order-by/order-by-type.enum";
import { ChColumnOutputDto } from "../output/ch-column-output.dto";

export class ChOrderByDto {
    orderByColumn: ChColumnOutputDto;
    orderBy: OrderByType = OrderByType.asc;
}